composer.json

```diff
@@ -150,8 +150,8 @@
     "umpirsky/country-list": "^2.0.6",
     "vrana/adminer": "^4.8.0",
     "vrana/jush": "~2.0.2",
-    "scheb/2fa-bundle": "^5.3",
-    "scheb/2fa-google-authenticator": "^5.3",
+    "scheb/2fa-bundle": "^5.3 || ^6.0",
+    "scheb/2fa-google-authenticator": "^5.3 || ^6.0",
```

Comment:
i was testing a little bit, it seems that we don't need to bump to 6 at all.
The 5.3 supports both since https://github.com/scheb/2fa/releases/tag/v5.10.0, while [6](
https://github.com/scheb/2fa/releases/tag/v6.0.0) is ONLY the new authenticator.

Reply:
Ohh, you are right. we can just implement the FC layer for new authenticator and then upgrade it to version 6 on Pimcore 11 as described in this issue https://github.com/pimcore/pimcore/issues/11662